pkgs/development/r-modules/wrapper-rstudio.nix

```diff
@@ -38,6 +59,9 @@ runCommand (rstudio.name + "-wrapper")
       echo -n $R_LIBS_SITE | sed -e 's/:/", "/g' >> $out/$fixLibsR
       echo -n "\"))" >> $out/$fixLibsR
       echo >> $out/$fixLibsR
+      makeWrapper "${R}/bin/R" "$out/bin/R" \
```

Comment:
On further thought, it would be good to reduce duplication. Could you drop all this extra wrapping for rstudio by using rWrapper or does rstudio also override the profile & environ variables?

Reply:
Just to check my understanding, are you suggesting that we just have the rstudio wrapper call the rWrapper? I'd also like to get rid of the duplication and force consistency between the IDE and CLI Rs, since the former often calls the latter in HPC applications.

Reply:
Yeah, if rstudio doesn't modify them then you should be able to change `buildInputs = [ R ...` to use `rWrapper` instead of `R` and pass through the two variables to rWrapper.

Reply:
I gave this a try in ae07707cbac48481557b8f9f2b20625d9655581c, but there are two issues:

1. RStudio overrides Renviron similarly to what it does with R_LIBS_SITE
2. Even if it worked, any change to rWrapper would then trigger an RStudio rebuild, which is wasteful

Is there another way to avoid the duplication, and keep the wrapper level insertion for RStudio? The rWrapper is most important because the professional RStudio product supports multiple versions of R, calling each directly and without overriding the files.